AUTHORS

```diff
@@ -1,13 +1,114 @@
+Alex Glikson <junjiez@microsoft.com>
+Ameer Deen <junjiez@microsoft.com>
+Amit Kumar <junjiez@microsoft.com>
+Andrey Moor <junjiez@microsoft.com>
+Anthony Chu <junjiez@microsoft.com>
+Anubhav Mishra <cpuguy83@gmail.com>
+Anubhav Mishra <junjiez@microsoft.com>
+Anubhav Mishra <mishra@hashicorp.com>Ahmed Sabbour <junjiez@microsoft.com>
+Anubhav Mishra <riabhatia@live.com>
+Ara Pulido <apulido@gmail.com>
+Ara Pulido <junjiez@microsoft.com>
+Ben Corrie <bcorrie@vmware.com>
+Ben Corrie <riabhatia@live.com>
+Bhargav Nookala <junjiez@microsoft.com>
+Bhargav Nookala <nooknb@gmail.com>
+Brian Goff <cpuguy83@gmail.com>
+Brian Goff <junjiez@microsoft.com>
+Brian Goff <riabhatia@live.com>
+Brian Ketelsen <alakriti@gmail.com>
 Brian Ketelsen <bketelsen@gmail.com>
+Bruno Miguel Custodio <brunomcustodio@gmail.com>
+Carlos Mendible <riabhatia@live.com>
+Chen Shou <junjiez@microsoft.com>
+Daniel Mueller <junjiez@microsoft.com>
+Deep Kapur <riabhatia@live.com>
+Don Bowman <junjiez@microsoft.com>
+Eddie Villalba <junjiez@microsoft.com>
+Eric Jadi <erjadi@microsoft.com>
+Eric Jadi <junjiez@microsoft.com>
 Erik St. Martin <alakriti@gmail.com>
+Fei Xu <cpuguy83@gmail.com>
+Fei Xu <junjiez@microsoft.com>
+Fei Xu <xufei40@huawei.com>
+Hector Fernandez <cpuguy83@gmail.com>
+Hector Fernandez <hfernandez@mesosphere.com>
+Hongbin Lu <cpuguy83@gmail.com>
+Jacob LeGrone <git@jacob.work>
+Jacob LeGrone <junjiez@microsoft.com>
+Jake Bjorke <junjiez@microsoft.com>
+Jeremy Rickard <cpuguy83@gmail.com>
+Jeremy Rickard <jeremy.rickard@microsoft.com>
+Jeremy Rickard <junjiez@microsoft.com>
```

Comment:
The name:email pairs here seem wrong all over the doc. I'm thinking this can happen because of commits that are co-authored, such as in PR merges.

Reply:
Yes I noticed, was going to check out AUTHORS in other repos to see if its expected or not.

Reply:
Ok, I changed the authors script to what we use in moby.